web/jest.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
         '^.+\\.(jsx|js)$': 'babel-jest',
         '^.+\\.svg$': '<rootDir>/jest/svg-transform.js',
     },
-    testRegex: '.src/.*.(/__tests__/.*|(\\.|/)(test|spec))\\.(tsx|jsx)?$',
+    testRegex: '.src/.*.(/__tests__/.*|(\\.|/)(test|spec))\\.(tsx|jsx|ts)?$',
```

Comment:
add js

Reply:
do we have test files with .js extension?

Reply:
we can have them